introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<p>Looks like a Jumpstart bucket and not a bucket just dedicated to built-in algorithms? I would make it more general instead (i.e. from a public S3 bucket)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>done.</p>